management/src/clusterctl/main.go

```diff
@@ -179,7 +196,11 @@ func newPostActioner(validateCb validateCallback, postCb postCallback) *postActi
 }
 
 func (npa *postActioner) procFlags(c *cli.Context) {
-	npa.extraVars = c.String("extra-vars")
+	npa.flags.extraVars = c.String("extra-vars")
```

Comment:
@mapuri need to check if `extraVarsFlag.Name` value will be `extra-vars` or `extra-vars, e`